src/transforms/groupby.js

```diff
@@ -35,14 +35,24 @@ exports.attributes = {
         ].join(' ')
     },
     style: {
```

Comment:
Maybe we should name this attributes `styles` instead? All (I think) `isLinkedToArray` attribute have plural names.